### PR TITLE
Addition of Newman-Conway Sequence in Mathematical Algorithms

### PR DESCRIPTION
**Pull request by mansiag**
**Fixes issue:**
N/A

**Changes:**
Added Newman-Conway Sequence in C++

### CODE REVIEW

File: code/mathematical_algorithms/src/newman_conway/Newman_Conway_Sequence.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#include <vector>
+
+int nthNewmanConwayNumber(int number)
+{
+    std::vector<int> arr;
+    arr.push_back(0);
+    arr.push_back(1);
+    arr.push_back(1);
+    for(size_t i = 3; i <= number; ++i)
+        arr.push_back(arr[arr[i-1]] + arr[i - arr[i-1]]);
```
Comment (YashovardhanSiramdas) [inline code/mathematical_algorithms/src/newman_conway/Newman_Conway_Sequence.cpp]: Could you give spaces between operands and operators?

Comment (mansiag) [reply code/mathematical_algorithms/src/newman_conway/Newman_Conway_Sequence.cpp]: Yeah sorry

**Comment by mansiag** [comment]
@YashovardhanSiramdas Can you please review the changes?

**Comment by mansiag** [comment]
Sorry, This has been accidentally closed.

File: code/mathematical_algorithms/src/newman_conway/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Newman - Conway Number
```
Comment (arnavb) [inline code/mathematical_algorithms/src/newman_conway/README.md]: Please follow our documentation guide. (I can't find a link to it)

File: code/mathematical_algorithms/src/newman_conway/newman_conway_sequence.cpp [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <vector>
+
+int nthNewmanConwayNumber(int number)
+{
+    std::vector<int> arr;
```
Comment (arnavb) [inline code/mathematical_algorithms/src/newman_conway/newman_conway_sequence.cpp]:
```C++
std::vector<int> arr {0, 1, 1};
```

File: code/mathematical_algorithms/src/newman_conway/newman_conway_sequence.cpp [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    arr.push_back(1);
+    for(size_t i = 3; i <= number; ++i)
+        arr.push_back(arr[arr[i - 1]] + arr[i - arr[i - 1]]);
+    return arr[number];
+}
+
+int main()
+{
+    int number;
+    std::cout << "Enter number:\n";
```
Comment (arnavb) [inline code/mathematical_algorithms/src/newman_conway/newman_conway_sequence.cpp]: Enter _a_ number: (No newline, just space)

File: code/mathematical_algorithms/src/newman_conway/newman_conway_sequence.cpp [hunk 5]
```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <vector>
+
+int nthNewmanConwayNumber(int number)
+{
+    std::vector<int> arr;
+    arr.push_back(0);
+    arr.push_back(1);
+    arr.push_back(1);
+    for(size_t i = 3; i <= number; ++i)
```
Comment (arnavb) [inline code/mathematical_algorithms/src/newman_conway/newman_conway_sequence.cpp]: `std::size_t`

**Comment by iattempt** [comment]
It is always happen. I have added feedback...

**Comment by arnavb** [comment]
@iattempt what do you mean?

File: code/mathematical_algorithms/src/newman_conway/README.md (hunk 2 above)
Comment (arnavb) [reply code/mathematical_algorithms/src/newman_conway/README.md]: Never mind, I found it: https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md. Please follow this guide for this document.

**Comment by iattempt** [comment]
@arnavb sorry for sentence. I mean that the close button is often clicked mistakenly, and I added feedback.

**Comment by mansiag** [comment]
@arnavb Is it alright?

File: code/mathematical_algorithms/src/newman_conway/README.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# Newman-Conway Number
```
Comment (arnavb) [inline code/mathematical_algorithms/src/newman_conway/README.md]: Make changes to follow [this guide](https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md). Otheriwse, it's good!

Comment (mansiag) [reply code/mathematical_algorithms/src/newman_conway/README.md]: Can you suggest me the changes?

Comment (arnavb) [reply code/mathematical_algorithms/src/newman_conway/README.md]: It's quite simple, just take the template, copy it into your Markdown, and fill in all required information.

**Comment by mansiag** [comment]
@arnavb @iattempt I have made some changes. Can you please review ?

**Comment by mansiag** [comment]
@YashovardhanSiramdas Can you please review the changes?

**Comment by mansiag** [comment]
@arnavb Thanks for the suggestion.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Comment by iattempt** [comment]
@mansiag Please create new PR which involve that fix these errors:
screen shot 2018-03-09 at 8 44 03 am

**Pull request merged**